backends/dart_dia/bin/server.dart

```diff
@@ -0,0 +1,55 @@
+import 'dart:convert';
+import 'dart:io';
+
+import 'package:dia/dia.dart';
+import 'package:dia_body/dia_body.dart';
+import 'package:dia_router/dia_router.dart';
+
+class ServerContext extends Context with Routing, ParsedBody {
+  ServerContext(super.request);
+
+  @override
+  String toString() {
+    var res = <String, String>{};
+    res['query'] = query.toString();
+    res['params'] = params.toString();
+    res['files'] = files.toString();
+    res['parsed'] = parsed.toString();
+
+    return res.toString();
+  }
+}
+
+void main(List<String> args) async {
+  // Use any available host or container IP (usually `0.0.0.0`).
+  final ip = "127.0.0.1";
+
+  final app = App<ServerContext>((req) => ServerContext(req));
+
+  final router = Router<ServerContext>('/');
+
+  router.get('/', (ctx, next) async {
+    ctx.body = json.encode({"response": 'Hello, World!'});
+  });
+  router.post('/echo', (ctx, next) async {
+    final map = ctx.parsed;
+    ctx.body = json.encode({"response": "Hello, ${map["name"]}!"});
+  });
+  router.post('/file_upload', (ctx, next) async {
+    final files = ctx.files;
+    ctx.body = files.length.toString();
```

Comment:
You have to return the size of content of the file. You will get file with name _benchmark_